src/cmd/build.js

```diff
@@ -220,7 +220,6 @@ export default async function build(
     showReadyMessage = true,
   }: BuildCmdOptions = {}
 ): Promise<ExtensionBuildResult> {
-
```

Comment:
Nit, not a big deal, but this change to src/cmd/build.js is unrelated and it would be better to not be part of this pull request.